### DISCUSSION TITLE
Help needed in being able to deploy this on cloud

### DISCUSSION THREAD

**Discussion by FlyingWedgeDefence** [AppFlowy-IO/AppFlowy#2206] [General help wanted] [answered]
I am following the instructions provided in the readme to install appflowy on a digital ocean instance. i am unable to run the docker commands and get the app up and running on my cloud instance and it always exits with errors. The errors occur from line 31. what do i do to fix this issue?

I believe general instructions are required to help users install the app on the cloud

**Comment by annieappflowy** [comment]
Hi @FlyingWedgeDefence , the cloud support is under development by @appflowy . It is scheduled to finish by the end of June. I will update you when it's ready. Thanks for checking out AppFlowy!

**Reply by Joso997** [reply]
is it ready?

**Reply by annieappflowy** [reply]
Can now be self-hosted with Supabase
Working on a docker solution

**Reply by Joso997** [reply]
Once the docker solution is ready (I would prefer a solution for digitalocean droplet too) will we be able to just transfer our local database to the cloud one?

**Reply by annieappflowy** [reply]
Yes

**Reply by annieappflowy** [reply, marked as answer]
We now support self-hosting AppFlowy with AppFlowy Cloud: https://docs.appflowy.io/docs/guides/appflowy/self-hosting-appflowy
@Joso997 , give it a try and let us know how you think

**Comment by annieappflowy** [comment]
> I am following the instructions provided in the readme to install appflowy on a digital ocean instance. i am unable to run the docker commands and get the app up and running on my cloud instance and it always exits with errors. The errors occur from line 31. what do i do to fix this issue?
> 
> I believe general instructions are required to help users install the app on the cloud

https://docs.appflowy.io/docs/guides/appflowy/self-hosting-appflowy
